dspace-api/src/main/java/org/dspace/statistics/SolrLoggerServiceImpl.java

```diff
@@ -213,14 +213,20 @@ public void post(DSpaceObject dspaceObject, HttpServletRequest request,
     @Override
     public void postView(DSpaceObject dspaceObject, HttpServletRequest request,
                          EPerson currentUser) {
-        if (solr == null) {
+        postView(dspaceObject, request, currentUser, null);
+    }
+
+    @Override
+    public void postView(DSpaceObject dspaceObject, HttpServletRequest request,
+                         EPerson currentUser, String referrer) {
+        if (solr == null || locationService == null) {
```

Comment:
locationService is no longer required after https://github.com/DSpace/DSpace/pull/8571.